src/project-tests/survey-form-tests.js

```diff
@@ -116,6 +116,13 @@ export default function createSurveyFormTests() {
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
                 var dropdown = document.getElementById('dropdown');
+				var dropdownid = dropdown.getAttribute('id');
```

Comment:
Using the string instead of a variable works fine for me. My problem was the getElementById('dropdown'), followed by getAttribute('id'). Obviously we don't need to get the id when we just specified it. ;-)